src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -25,10 +25,11 @@
 
 package org.spongepowered.api.world.extent;
 
-import com.flowpowered.math.vector.Vector3i;
 import org.spongepowered.api.block.BlockLoc;
 import org.spongepowered.api.world.weather.WeatherUniverse;
 
+import com.flowpowered.math.vector.Vector3i;
```

Comment:
@Minecrell Aaaargh... why does this keep happening? My eclipse formatter is set to only format lines I edit!